consumption.tex

```diff
@@ -44,6 +44,10 @@ \subsection{Bootstraping Mechanism}
 \subsection{Verifying Block Connection}\label{section:block-connection}
 We now shift our attention to the problem of verifying whether a block really belongs in the best source chain. We provide multiple ways of implementing the aforementioned \textsf{is-in-best-chain} method.
 
+\noindent
+\textbf{Direct observation.}
+Miners connect to the source blockchain network and have access to the best source chain. A miner can thus evaluate if a block is included in that chain. This mechanism does not provide miner-isolation. It is adopted by Counterparty.
```

Comment:
Do we mention the property of miner-isolation before?

Reply:
I introduce it on another PR at the top of the section.